### PR TITLE
CIV-6431 Submit Debt Respite

### PR DESCRIPTION
**Pull request by kenneth-hmcts**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-6431
https://tools.hmcts.net/jira/browse/CIV-6381

### Change description ###
This PR contains changes on the Debt Respite from Citizen UI Side

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by TatsyMaslova — DISMISSED** [review]

**Review by laperla11 — DISMISSED** [review]

**Review by LeonardoPalmeiro — DISMISSED** [review]
LGTM

File: src/main/services/translation/breathingSpace/ccdTranslation.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import {CCDClaim} from 'models/civilClaimResponse';
+import {BreathingSpace} from 'models/breathingSpace';
+import {toCCDBreathingSpaceStartInfo} from 'services/translation/breathingSpace/convertToCCDBreathingSpaceStrartInfo';
```
Comment (PaulH-HMCTS) [inline src/main/services/translation/breathingSpace/ccdTranslation.ts:3]: Typo on file name
```suggestion
import {toCCDBreathingSpaceStartInfo} from 'services/translation/breathingSpace/convertToCCDBreathingSpaceStartInfo';
```

File: src/test/unit/services/features/breathingSpace/submission/submitBreathingSpace.test.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
+import * as draftStoreService from 'modules/draft-store/draftStoreService';
+import * as ccdTranslationService from 'services/translation/breathingSpace/ccdTranslation';
+import {Claim} from 'common/models/claim';
+import * as requestModels from 'common/models/AppRequest';
+import nock from 'nock';
+import config from 'config';
+import {TestMessages} from '../../../../../utils/errorMessageTestConstants';
+import {submitBreathingSpace} from 'services/features/breathingSpace/submission/submitBreathingSpace';
+import {ClaimDetails} from 'form/models/claim/details/claimDetails';
+jest.mock('../../../../../../main/modules/draft-store');
```
Comment (PaulH-HMCTS) [inline src/test/unit/services/features/breathingSpace/submission/submitBreathingSpace.test.ts:10]: Add some line separators here to separate the mocks, imports and consts

File: src/test/unit/services/translation/breathingSpace/convertToCCDBreathingSpaceStrartInfo.test.ts [hunk 3]
```diff
@@ -0,0 +1,93 @@
+import {BreathingSpace} from 'common/models/breathingSpace';
+import {DebtRespiteStartDate} from 'common/models/breathingSpace/debtRespiteStartDate';
+import {toCCDBreathingSpaceStartInfo} from 'services/translation/breathingSpace/convertToCCDBreathingSpaceStrartInfo';
```
Comment (PaulH-HMCTS) [inline src/test/unit/services/translation/breathingSpace/convertToCCDBreathingSpaceStrartInfo.test.ts:3]: Typo here and this file name
```suggestion
import {toCCDBreathingSpaceStartInfo} from 'services/translation/breathingSpace/convertToCCDBreathingSpaceStartInfo';
```

File: src/test/unit/services/translation/claim/ccdTransaltion.test.ts [hunk 4]
```diff
@@ -9,7 +9,7 @@ describe('translate draft claim to ccd version', () => {
     //Given
     const claim = new Claim();
     //When
-    const ccdClaim = translateDraftClaimToCCD(claim);
+    const ccdClaim = translateDraftClaimToCCD(claim, undefined);
```
Comment (PaulH-HMCTS) [inline src/test/unit/services/translation/claim/ccdTransaltion.test.ts:12]: As others mentioned in the code walkthrough, you shouldnt need this once you merge Leo's changes in

File: src/main/services/translation/breathingSpace/ccdTranslation.ts (hunk 1 above)
Comment (kenneth-hmcts) [reply src/main/services/translation/breathingSpace/ccdTranslation.ts]: Changes made as suggested above

File: src/test/unit/services/features/breathingSpace/submission/submitBreathingSpace.test.ts (hunk 2 above)
Comment (kenneth-hmcts) [reply src/test/unit/services/features/breathingSpace/submission/submitBreathingSpace.test.ts:11]: Changes made as suggested above

File: src/test/unit/services/translation/breathingSpace/convertToCCDBreathingSpaceStrartInfo.test.ts (hunk 3 above)
Comment (kenneth-hmcts) [reply src/test/unit/services/translation/breathingSpace/convertToCCDBreathingSpaceStrartInfo.test.ts]: Changes made as suggested above

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by TatsyMaslova — DISMISSED** [review]

**Review by PaulH-HMCTS — APPROVED** [review]

**Pull request closed without merging**